extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -511,6 +510,37 @@ export class AmpStoryEmbeddedComponent {
         });
   }
 
+  /**
+   * Attaches listeners that listen for UI updates.
+   * @private
+   */
+  attachUIListeners_() {
```

Comment:
Nit: we mostly use `initializeListeners` and sometimes `startListening` in the codebase